backend/src/test/java/codezap/template/repository/SourceCodeRepositoryTest.java

```diff
@@ -0,0 +1,209 @@
+            var id = 100L;
+            assertThatThrownBy(() -> sut.fetchById(id))
+                    .isInstanceOf(CodeZapException.class)
+                    .hasMessageContaining("식별자 " + id + "에 해당하는 소스 코드가 존재하지 않습니다.");
+        }
+    }
+
+    @Nested
+    @DisplayName("템플릿으로 전체 소스코드 조회")
+    class FindAllByTemplate {
+        @Test
+        @DisplayName("성공: 템플릿으로 전체 소스코드 조회")
+        void findAllByTemplateSuccess() {
+            var member1 = memberRepository.save(MemberFixture.getFirstMember());
+            var category1 = categoryRepository.save(CategoryFixture.getFirstCategory());
+            var template1 = templateRepository.save(new Template(member1, "Template 1", "Description 1", category1));
+            var template2 = templateRepository.save(new Template(member1, "Template 2", "Description 2", category1));
+            var sourceCode1 = new SourceCode(template1, "SourceCode 1", "Content 1", 1);
+            var sourceCode2 = new SourceCode(template1, "SourceCode 2", "Content 2", 2);
+            var sourceCode3 = new SourceCode(template2, "SourceCode 3", "Content 3", 1);
+            sut.save(sourceCode1);
+            sut.save(sourceCode2);
+            sut.save(sourceCode3);
+
+            var result = sut.findAllByTemplate(template1);
+
+            assertThat(result).hasSize(2)
+                    .containsExactly(sourceCode1, sourceCode2);
+        }
+
+        @Test
+        @DisplayName("성공: 소스코드가 없는 템플릿으로 조회하는 경우 빈 리스트가 반환된다.")
+        void findAllByTemplateSuccessWithNoSourceCodeTemplate() {
+            var member1 = memberRepository.save(MemberFixture.getFirstMember());
+            var category1 = categoryRepository.save(CategoryFixture.getFirstCategory());
+            var template1 = templateRepository.save(new Template(member1, "Template 1", "Description 1", category1));
+
+            var result = sut.findAllByTemplate(template1);
+
+            assertThat(result).isEmpty();
+        }
+    }
+
+    @Nested
+    @DisplayName("템플릿과 소스코드 순서로 조회")
+    class FetchByTemplateAndOrdinal {
+        @Test
+        @DisplayName("성공: 템플릿과 소스코드 순서로 조회")
+        void fetchByTemplateAndOrdinalSuccess() {
+            var member1 = memberRepository.save(MemberFixture.getFirstMember());
+            var category1 = categoryRepository.save(CategoryFixture.getFirstCategory());
+            var template1 = templateRepository.save(new Template(member1, "Template 1", "Description 1", category1));
+            var template2 = templateRepository.save(new Template(member1, "Template 2", "Description 2", category1));
+            var sourceCode1 = new SourceCode(template1, "SourceCode 1", "Content 1", 1);
+            var sourceCode2 = new SourceCode(template1, "SourceCode 2", "Content 2", 2);
+            var sourceCode3 = new SourceCode(template2, "SourceCode 3", "Content 3", 1);
+            sut.save(sourceCode1);
+            sut.save(sourceCode2);
+            sut.save(sourceCode3);
```

Comment:
```suggestion
            var sourceCode1 = sut.save(new SourceCode(template1, "SourceCode 1", "Content 1", 1));
            var sourceCode2 = sut.save(new SourceCode(template1, "SourceCode 2", "Content 2", 2));
            var sourceCode3 = sut.save(new SourceCode(template2, "SourceCode 3", "Content 3", 1));
```

이 부분도 바로 위 라인처럼 작성할 수 있지 않을까요?!
라인이 줄어들 것 같아요~!